src/test/java/org/fundacionjala/coding/maria/Kata5Test.java

```diff
@@ -0,0 +1,78 @@
+package org.fundacionjala.coding.maria;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Test to kata5.
+ */
+public class Kata5Test {
+    /**
+     * Test testComplementaryDNA.
+     */
+    @Test
+    public void testComplementaryDNA() {
+        Kata5 dna = new Kata5();
+        assertEquals("TAACG", dna.makeComplement("ATTGC"));
+    }
+    /**
+     * Test test01.
+     */
+    @Test
+    public void test01() {
+        Kata5 dna = new Kata5();
+        assertEquals("TTTT", dna.makeComplement("AAAA"));
+    }
+    /**
+     * Test test02.
+     */
+    @Test
+    public void test02() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "testComplementaryDNA" on line 15. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)